src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -21,7 +21,9 @@ public sealed partial class JsonSerializerOptions
         // The global list of built-in converters that override CanConvert().
         private static readonly JsonConverter[] s_defaultFactoryConverters = new JsonConverter[]
         {
-            // Nullable converter should always be first since it forwards to any nullable type.
+            // Check for black-listed types.
```

Comment:
This is going to hit policheck.

```suggestion
            // Check for disallowed types.
```